player/public/coreplugins/org.sunbird.player.endpage-1.1/renderer/endpageApp.js

```diff
@@ -45,7 +45,11 @@ endPage.controller("endPageController", function($scope, $rootScope, $state,$ele
     };
    
     $scope.replayContent = function() {
-        EventBus.dispatch("event:openUserSwitchingModal", {'logGEEvent': $scope.pluginInstance._isAvailable});
+        if(isbrowserpreview) {
+            $scope.replayCallback();
+        }else {
+            EventBus.dispatch("event:openUserSwitchingModal", {'logGEEvent': $scope.pluginInstance._isAvailable});
```

Comment:
why are using eventBus ? please use API

Reply:
EkstepRendererAPI.dispatched method is undefined here